src/fontMetrics.js

```diff
@@ -106,7 +106,10 @@ var metrics = {
             return sigma21ScriptScript;
         }
         throw new Error("Unexpected style size: " + style.size);
-    }
+    },
+
+    // \nulldelimiterspace=1.2pt, pg. 344 TeXbook
+    nulldelimiterspace: 1.2 / ptPerEm
```

Comment:
In #295 I've added a `makeNullDelimiter` function which adds all the classes to change back to textstyle size5 so that the width given relative to that will actually be correct even if the surrounding text is scaled.